src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Errors/ErrorCode.cs

```diff
@@ -64,6 +64,7 @@ internal enum ErrorCode
         ERR_NamedArgumentUsedInPositional = 1744,
         ERR_BadNamedArgumentForDelegateInvoke = 1746,
         ERR_NonInvocableMemberCalled = 1955,
-        ERR_BadNonTrailingNamedArgument = 8323
+        ERR_BadNonTrailingNamedArgument = 8323,
+        ERR_DynamicBindingComUnsupported,
```

Comment:
>ERR_DynamicBindingComUnsupported [](start = 8, length = 32)

Are these `ErrorCode` values exposed in the Exception?

If so, consider using an explicit value that is not already used by the C# compiler (see [https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Errors/ErrorCode.cs](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Errors/ErrorCode.cs)). Perhaps 8365 for instance.